### PR TITLE
Usagov 1109 update readme based on environment setup experience

### PR DESCRIPTION
**Pull request by YaritzaGarcia**
## Jira Task
[https://cm-jira.usa.gov/browse/USAGOV-1109](https://cm-jira.usa.gov/browse/USAGOV-1109)

## Description
To improve the onboarding process, I have added:
- Table of Content and "back to top" feature to facilitate the navigation
- Prerequisites section listing all necessary tools
- Step-by-step setup instructions using plain language
- Initialization section
- Media files Setup section
- How to re-run the website section

## Checklist for the Developer

- [x] A link to the JIRA ticket has been added above.
- [x] The JIRA ticket identifies the desired result of this change.
- [x] The JIRA ticket contains clear acceptance criteria.
- [x] The JIRA ticket contains clear testing steps.
- [x] The JIRA ticket contains a description of "Done".
- [x] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,207 +2,436 @@
...
+
+* [Homebrew](https://docs.brew.sh/Installation)
+* [Git](https://formulae.brew.sh/formula/git#default)
+* [Php](https://formulae.brew.sh/formula/php#default)
+* [Composer](https://formulae.brew.sh/formula/composer#default)
+* [Docket Desktop](https://docs.docker.com/desktop/install/mac-install/)
+* [Drupal](https://www.digitalocean.com/community/tutorials/how-to-develop-a-drupal-9-website-on-your-local-machine-using-docker-and-ddev#step-1-mdash-installing-ddev)
+* [jq](https://formulae.brew.sh/formula/jq#default)
+* [npm](https://radixweb.com/blog/installing-npm-and-nodejs-on-windows-and-mac)
+* [Visual Studio Code](https://code.visualstudio.com/download)
```
Comment (jacobaaronyeager) [inline README.md]: VS Code is not _required_, just a code editor/IDE of some kind, VSCode is just the most popular right now. :)

File: README.md [hunk 2]
```diff
@@ -2,207 +2,436 @@
...
+1. Download the latest SQL database available from [our Google Drive](https://drive.google.com/drive/folders/1zVDr7dxzIa3tPsdxCb0FOXNvIFz96dNx?usp=sharing).
+
+
+2. Unzip the file and move the .sql file directly into the **root** of your USAgov project folder.
+
+
+3. Open a new terminal and go to your USAgov project folder.
+
+
+4. Run this command to populate the database from the SQL file you downloaded:
```
Comment (jacobaaronyeager) [inline README.md]: Does there need to be a step here about renaming the SQL file or am I crazy?

**Review by jacobaaronyeager — CHANGES_REQUESTED** [review]
Only thing I think that should change is noting that VSCode isn't the requirement, just an IDE, but linking to VSCode is great, as it's free and popular.

File: README.md (hunk 1 above)
Comment (YaritzaGarcia) [reply README.md]: Thank you for pointing that out. I have made the change.

File: README.md (hunk 2 above)
Comment (YaritzaGarcia) [reply README.md]: You're right! You don't need to change the title but if you don't change it, the command is different. I change that section. Thank you!

**Review by jacobaaronyeager — APPROVED** [review]

**Pull request merged**